### ISSUE TITLE
 Issue on 02_lights_eda.ipynb

### ISSUE THREAD

**Issue by mnitin59** [thinkingmachines/ph-poverty-mapping#15]
I have encountered a few issues while working on the notebook file. 

In Night Time Lights (NTL) Dataset EDA Section 
-> # Get sum of population per cluster Cell 

There is one error in which nightlights_unstack.csv file doesn't create with a pop_sum column where it's giving 'no object found' error, while in the nightlights.csv has a pop_sum column 
>> Need suggestion whether to use unstack file or nightlight file for population clustering. 

In Load DHS Dataset Section 
-> merging nightlights and indicators data showing this error 
![DHS Merge Issue](https://user-images.githubusercontent.com/15194333/58223214-6fdd7080-7d36-11e9-9ad5-16dbfc88da9f.png)

In Correlations between Average Nighttime Lights and Socioeconomic Indicators Section 
-> Data is not loading properly 
![Coorelation Issue ](https://user-images.githubusercontent.com/15194333/58223263-956a7a00-7d36-11e9-858f-20a70a790583.png)

In Balancing Nighttime Lights Intensity Levels
-> It requires report.csv files. 

Please let me know the following solutions on this issue. 
I request you to provide data files as there are lots of missing data while executing the program. In other case generating the file, it's getting little confusion on plotting data properly or data getting mismatch for further analysis its taking lots of time for getting into the processing. 
So if possible can share the data files to mnitin59@gmail.com till osm model. 
Further would share research work which would be helpful for both.

**Comment by ibtingzon** [comment]
@mnitin59 Thanks for catching this. Fixed this buy in #16. 
I added that bit of code as an afterthought, just to check the correlation between total population per cluster and cluster wealth. I realized I hadn't cleaned it properly. The graph is also properly labelled now. 

Also added the OSM files here in the [data directory](https://github.com/thinkingmachines/ph-poverty-mapping/tree/master/data). Let us know if you have any additional questions. Thanks!

**Comment by ibtingzon** [comment]
To generate `report.csv`, you'll need to uncomment the following section and download the images. 
Note, you'll need to obtain an API key and a secret key to download Google Static Maps images.  [More information on that here.](https://developers.google.com/maps/documentation/maps-static/intro)
We would love to share the images with you, but unfortunately, we cannot redistribute them under the [Google Static Maps Terms of Services](https://developers.google.com/maps/terms-20180207). 
Also, please take note of the cost of obtaining images; downloading 1000 images costs around ~$1.6. 
```
# Uncomment to download data
os.environ['GEO_AI_API_KEY'] = "INSERT API KEY HERE"
os.environ['GEO_AI_SECRET_KEY'] = "INSERT SECRET KEY HERE"

import data_download
data_download.get_satellite_images_with_labels(nightlights_bins_file, satellite_images_dir)
```
